testsuite/tests/project_support/test.sh

```diff
@@ -0,0 +1,8 @@
+# Reduce in directory dir1
+(cd dir1 ; $ADAREDUCER p.gpr oracle.sh > /dev/null)
+
+# Check that dir1/hello.adb has been minimized
+grep "with P" dir1/hello.adb
+
+# Check that dir2/p.adb has been suppressed
+[ ! -e dir2/p.adb ] || echo "oh no dir2/p.adb still exists"
```

Comment:
Or ` [-e dir2/p.adb] && echo "oh no"` seems more direct, no?

Reply:
I started with this, unfortunately the default e3 test driver looks at the exit status of the script - so if we do `- e` it'll return an error code, therefore failing the test.